lib/object-mask.js

```diff
@@ -396,34 +405,30 @@ function invert(mask) {
 	return result;
 }
 
-// Throws on (deep) attempt to subtract collection from scalar
 function subtract(a, b) {
 	// base cases:
+	if (a === true) return invert(b);
+	if (a === false) return false;
+	if (!a) return invert(b);
```

Comment:
`a` can also be (and often is) `undefined`, which needs to be handled differently from `false`.